services/question-service/src/entry-points/api/routes.ts

```diff
@@ -30,7 +30,11 @@ export default function defineRoutes(expressApp: express.Application) {
         difficulty: req.query.difficulty as Difficulty,
         categories: req.query.categories as Category[],
       });
-      res.json(response);
+      if (!req.query.getOne || !response) {
+        res.json(response);
+      } else {
+        res.json(response[Math.floor(Math.random() * response.length)]);
+      }
```

Comment:
imo this could probably be like `GET /questions/random` ah, since its not really a filter upon the questions

Reply:
I'll merge this first then make this change in a future PR :D